Engine/ResourceManagement/Importer/TextureImporter.cpp

```diff
@@ -28,31 +29,18 @@ std::pair<bool, std::string> TextureImporter::Import(const File & file) const
 	}
 
 	App->filesystem->MakeDirectory(LIBRARY_TEXTURES_FOLDER);
-	//Bound image
+	
+	std::string output_file;
 
-	ILuint image;
-	ilGenImages(1, &image);
-	ilBindImage(image);
-	int width, height;
-	ILubyte * save_data = LoadImageData(file.file_path.c_str(), IL_RGBA, width, height);
-	//Get new Name
-
-	std::string texture_name_no_extension = file.filename.substr(0, file.filename.find_last_of("."));
-	std::string output_file = LIBRARY_TEXTURES_FOLDER + "/" + texture_name_no_extension + ".dds";
-
-	//Save data
-	ILuint size;
-
-	ilSetInteger(IL_DXTC_FORMAT, IL_DXT5);// To pick a specific DXT compression use
-	size = ilSaveL(IL_DDS, NULL, 0); // Get the size of the data buffer
-	if (size > 0) {
-
-		if (ilSaveL(IL_DDS, save_data, size) > 0) // Save to buffer with the ilSaveIL function
-		{
-			App->filesystem->Save(output_file.c_str(), (unsigned int *)save_data, size);
-		}
-		ilDeleteImages(1, &image);
+	if (file.filename.find("_normal") != std::string::npos)
+	{
+		output_file = ImportToTGA(file);
+	}
+	else 
+	{
+		output_file = ImportToDDS(file);
```

Comment:
What if the file ends with xxxNormal.xxx or xxxnormal.xxx or another variation? should we consider this or tell the artists to send it as xxxx.normal?

Reply:
Well in the future the idea is to have importing options, so they can click a checkbox for this instead of relaying in the name. This is only a  mvp so you can start working.

Reply:
I think it's better to handle it in the code, if its xxxNormals or xxxxnormals, but we will discuse it :D 
Thank you for your work!